MAINTAINERS_GUIDE.md

```diff
@@ -0,0 +1,90 @@
+
+This is an open-source project with an open design philosophy.
+This means that the repository is the source of truth for EVERY aspect of the project, including its philosophy, design, roadmap and APIs.
+*If it's part of the project, it's in the repo.
+It's in the repo, it's part of the project.*
+
+As a result, all decisions can be expressed as changes to the repository.
+An spec change is a change to the specification.
+An implementation change is a change to the source code.
+A philosophy change is a change to the philosophy manifesto.
+And so on.
+
+All decisions affecting this project, big and small, follow the same 3 steps:
+
+* Step 1: Open a pull request. Anyone can do this.
+
+* Step 2: Discuss the pull request. Anyone can do this.
+
+* Step 3: Accept (`LGTM`) or refuse a pull request. The relevant maintainers do this (see below "Who decides what?")
+
+*I'm a maintainer, should I make pull requests too?*
+
+Yes.
+Nobody should ever push to the `main` branch directly.
+All changes should be made through a pull request.
+
+Bigger changes (that may span projects): an [OCI Working Group](https://github.com/opencontainers/tob/blob/main/CHARTER.md) may be needed.
+
+## Who decides what?
+
+All decisions are pull requests, and the relevant maintainers make decisions by accepting or refusing the pull request.
+Review and acceptance by anyone is denoted by adding a comment in the pull request: `LGTM`.
+However, only currently listed `MAINTAINERS` are counted towards the required **two LGTMs**.
+
+Overall the maintainer system works because of mutual respect across the maintainers of the project.
+The maintainers trust one another to make decisions in the best interests of the project.
+Sometimes maintainers can disagree and this is part of a healthy project to represent the point of views of various people.
+
+The maintainer system is built on trust, if there is a conflict the maintainers do not agree on it can be brought to the [technical oversight board](https://github.com/opencontainers/tob) for resolution.
+It is expected that this would be a very exceptional event.
+
+## How are maintainers added?
+
+The best maintainers have a vested interest in the project.
+Maintainers are first and foremost contributors that have shown they are committed to the long term success of the project.
+Contributors wanting to become maintainers are expected to be deeply involved in contributing code, pull request review, and triage of issues in the project for more than two months.
+
+Just contributing does not make you a maintainer, it is about building trust with the current maintainers of the project and being a person that they can depend on and trust to make decisions in the best interest of the project.
+The final vote to add a new maintainer should be approved by over **66% (2/3) of the current maintainers**.
+The voting period is **five business days** on the Pull Request to add the new maintainer.
```

Comment:
Should this say "at least" five business days, so that votes after the five days are still counted if it takes that long to get enough maintainer responses?

Reply:
yeah fair